src/sage/databases/cubic_hecke_db.py

```diff
@@ -215,7 +215,7 @@ def demo_version(self):
         return self._demo
 
     # --------------------------------------------------------------------------
-    # read from an sobj-file obtained from Ivan Marin's database
+    # read from a sobj-file obtained from Ivan Marin's database
```

Comment:
I think "an" is correct here because "sobj" would be pronounced "ess object"

same a few times below

Reply:
I thought it would be a "sob-jay".

Reply:
I agree with @mkoeppe that it should be "an". The pronunciation is ambiguous, but "s-o-b-j" or "s-object" both seem natural.

Reply:
```suggestion
    # read from an sobj-file obtained from Ivan Marin's database
```